argo-cd-apps/base/member/infra-deployments/kyverno/kyverno.yaml

```
@@ -0,0 +1,47 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: kyverno
+  labels:
+    noSourceTransform: "true"
+spec:
+  generators:
+  - merge:
+      mergeKeys:
+        - nameNormalized
+      generators:
+        - clusters:
+            values:
+              sourceRoot: components/kyverno
+              environment: staging
+              clusterDir: ""
+        - list:
+            elements: []
+  template:
+    metadata:
+      name: kyverno-{{nameNormalized}}
+    spec:
+      project: default
+      sources:
+          # Chart from upstream Kyverno
+        - chart: kyverno
+          repoURL: https://kyverno.github.io/kyverno/
+          targetRevision: 1.13
```

It seems to me this value should be the ChartVersion

cf. https://github.com/kyverno/kyverno/tree/release-1.13/charts/kyverno#installing-the-chart

```suggestion
          targetRevision: 3.3.4
```

---

Fixed in ed5b523.